nexus3/install.sh

```diff
@@ -7,7 +7,7 @@ kubectl config set-context --current --namespace=$NEXUS_NS
 helm repo add sonatype https://sonatype.github.io/helm3-charts/
 helm repo update
 
-helm install -f values.yaml --wait \
+helm install -f https://raw.githubusercontent.com/kameshsampath/kubernetes-marketplace/nexus3/nexus3/values.yaml --wait \
```

Comment:
don't use path from your repository, it is a predictable [at once it's gets merged 
https://raw.githubusercontent.com/civo/kubernetes-marketplace/master/nexus3/values.yaml